lms/templates/footer.html

```diff
@@ -83,3 +87,15 @@
     <link rel="stylesheet" type="text/css" href="${url}"></link>
   % endfor
 % endif
+
+<script>
+    $(document).ready(function(){
+        $.ajax({
```

Comment:
It does not make sense to hit db every time we have to load a page just to fill in FAQ title. Please override `footer.html` in philu's theme and name FAQ link to whatever needed.

Reply:
FAQ title can be changed through admin panel anytime by admin.